package.json

```diff
@@ -16,7 +16,7 @@
     "db:seed": "sequelize db:seed:all",
     "start": "node dist/index.js",
     "start:dev": "cross-env NODE_ENV=development DEBUG=dev nodemon --exec babel-node server",
-    "test": "nyc mocha --timeout 1000 -r esm --exit"
+    "test": "nyc mocha ./test/index.test.js --timeout 1000 -r esm --exit"
```

Comment:
ditto

Reply:
Since the team agreed on having an index file for every folder I decided to import all test files into the index file and make the test run from the index file